### DISCUSSION TITLE
编译报错是为什么啊

### DISCUSSION THREAD

**Discussion by sqwev** [whutug/whu-thesis#78] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/50485352/125113679-8a902080-e11b-11eb-91d9-3e230480a9ff.png)

**Comment by sikouhjw** [comment, marked as answer]
请使用 texlive 2021。话说没有检测版本的代码吗？ @tanukihee

**Reply by tanukihee** [reply]
没有，反正跑不通更新就是了，检查了也没用（逃）

最低保证是 overleaf，overleaf 能跑通就行

**Reply by sqwev** [reply]
感谢您的回答，问题解决了

**Comment by tanukihee** [comment]
见 #35（以及一连串的跟 #35 一样的，标了 duplicate 的反馈

但凡拿 `\tl_if_eq` 搜一搜就有了嘛（
